pkg/model/components/apiserver.go

```diff
@@ -133,8 +133,6 @@ func (b *KubeAPIServerOptionsBuilder) BuildOptions(o interface{}) error {
 
 	c.AllowPrivileged = fi.Bool(true)
 	c.ServiceClusterIPRange = clusterSpec.ServiceClusterIPRange
-	c.EtcdServers = []string{"http://127.0.0.1:4001"}
-	c.EtcdServersOverrides = []string{"/events#http://127.0.0.1:4002"}
```

Comment:
I would prefer this logic to remain in model and in NodeUp to just use whatever was set earlier. What do you think @codablock?

Reply:
I'm not sure I understand. You want to leave these two lines as they were? If this is what you mean, then it becomes strange in `buildPod`, as I'd have to clear `EtcdServersOverrides` in case of `! hasEventsEtcdCluster`, which IMHO would be strange and not immediately clear why this is done (for example, it was a surprise for me that `EtcdServersOverrides` was still set, even though I skipped the assignment in `buildPod` when I started trying out things).

Reply:
I tried to explain with code what I meant. Easier than words :).